extruct/jsonld.py

```diff
@@ -34,7 +34,7 @@ def _extract_items(self, node):
             data = json.loads(script, strict=False)
         except ValueError:
             # sometimes JSON-decoding errors are due to leading HTML or JavaScript comments
-            data = json.loads(
+            data = jstyleson.loads(
                 HTML_OR_JS_COMMENTLINE.sub('', script), strict=False)
```

Comment:
Is `HTML_OR_JS_COMMENTLINE` still needed here?

Reply:
Actually, It is not needed. I could remove it later.